src/styles/card.scss

```diff
@@ -0,0 +1,111 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $white-color;
```

Comment:
Default
```suggestion
```

Reply:
I didn't understand it either. How exactly do you fix this?